examples/jsm/loaders/KTX2Loader.js

```diff
@@ -126,11 +126,11 @@ class KTX2Loader extends Loader {
 
 			this.workerConfig = {
 				astcSupported: renderer.hasFeature( 'texture-compression-astc' ),
-				etc1Supported: false,
+				etc1Supported: renderer.hasFeature( 'texture-compression-etc1' ),
 				etc2Supported: renderer.hasFeature( 'texture-compression-etc2' ),
 				dxtSupported: renderer.hasFeature( 'texture-compression-bc' ),
-				bptcSupported: false,
-				pvrtcSupported: false
+				bptcSupported: renderer.hasFeature( 'texture-compression-bptc' ),
+				pvrtcSupported: renderer.hasFeature( 'texture-compression-pvrtc' )
```

Comment:
What is the reason for removing the `false` flags here? 

@donmccurdy I guess these flags for ETC1, BPTC and PVRTC were set to `false` on purpose.

Reply:
From the [official WebGPU specs](https://gpuweb.github.io/gpuweb/#gpufeaturename) `pvrtc` and `bptc` are not available so I would guess that's why. Nevertheless we need them in the WebGL context.